src/OpenIddict.Core/Views/Shared/Authorize.cshtml

```diff
@@ -1,14 +1,13 @@
 ﻿@using AspNet.Security.OpenIdConnect.Extensions
 @using Microsoft.IdentityModel.Protocols.OpenIdConnect
-@using OpenIddict.Models
 
-@model Tuple<OpenIdConnectMessage, Application>
+@model Tuple<OpenIdConnectMessage, string>
```

Comment:
Maybe a `OpenIddictConnectResult` is better suited than a ViewModel, like IdentityResult and SignInResult but I don't know where `request` is coming from at controller level so I leave this for you to decide.